config.js

```diff
@@ -1,3 +1,7 @@
+import { isOfficialBuild } from './app/constants/environment';
+
+const BugSnagApiKey = (isOfficialBuild) ? '0e62781637138bc11e6ad3ca14c03899' : '';
```

Comment:
where this key came from?

Reply:
from [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/609bf89e02984b186827af287328e97e8a969bb5/android/gradle.properties#L34)